File: TiaCodegen/Commands/NCoil.cs
using TiaCodegen.Commands.Coils;
using TiaCodegen.Commands.Signals;
using TiaCodegen.Interfaces;

namespace TiaCodegen.Commands
{
    public class NCoil : BaseNPCoil
    {
        public NCoil(Signal signal, Signal helpSignal, IOperationOrSignal op = null) : base(signal, helpSignal, op)
        { }
    }
}

File: TiaCodegen-ts/src/Commands/NCoil.ts
import { IOperationOrSignal } from '../Interfaces/IOperationOrSignal.js';
import { Signal } from './Signals/Signal.js';
import { BaseNPCoil } from './Coils/BaseNPCoil.js';

export class NCoil extends BaseNPCoil {
    constructor(signal: Signal, helpSignal: Signal, op: IOperationOrSignal | null = null) {
        super(signal, helpSignal, op);
    }
}
